src/sentry/issues/ingest.py

```diff
@@ -105,7 +105,7 @@ def _create_issue_kwargs(
         "type": occurrence.type.type_id,
         "first_release": release,
         "data": materialize_metadata(occurrence, event),
-        "priority": occurrence.initial_issue_priority,
+        "priority": occurrence.initial_issue_priority or occurrence.type.default_priority,
```

Comment:
This is technically superfluous, but we call `save_issue_occurrence` in lots of tests which skips setting the default priority. Worth considering if we should disallow that if the intended pipeline for occurrences is via `_process_message`